cmake/CreateBazelConfig.cmake

```diff
@@ -124,7 +124,14 @@ function (export_list_to_bazel filename)
     foreach (VAR ${_EXPORT_LIST_TO_BAZEL_OPT_UNPARSED_ARGUMENTS})
         file(APPEND "${filename}" "\n${VAR} = [\n")
         foreach (item ${${VAR}})
-            file(APPEND "${filename}" "    \"${item}\",\n")
+            # While cmake needs compute as a feature, it confuses bazel.
+            if (${item} STREQUAL "compute")
```

Comment:
This seems like the wrong place to implement this functionality.  This function is a general-purpose "export a list to a `.bzl` file".  It seems it should be called with a list that contains the regular features and all the values from `GOOGLE_CLOUD_CPP_COMPUTE_LIBRARIES` .